samples/file-encryptor/enclave/openssl_src/encryptor.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openssl/aes.h>
+#include <openssl/evp.h>
+#include <string.h>
+
+#include "aes_context.h" // Must be included before common/encryptor.h
+#include "common/encryptor.h"
+#include "common/trace.h"
+
+ecall_dispatcher::ecall_dispatcher() : m_encrypt(true), m_header(NULL)
```

Comment:
use nullptr instead of NULL in files that are C++ specific (e.g., using std::string)

Reply:
changed instances of NULL to nullptr.